test/distributed/_composable/fsdp/test_fully_shard_compile.py

```diff
@@ -99,10 +103,18 @@ def patched_trace_rules_check(*args, **kwargs):
 class TestFullyShardCompile(FSDPTest):
     fake_pg = not at_least_x_gpu(2)
 
+    # TODO: this setting overrides the setting of base class FSDPTest, which uses `DEVICE_COUNT`.
+    # Further inspection is needed to see if it is compatible with other codes in the base class.
     @property
     def world_size(self) -> int:
         return 2
 
+    # Tests in this class requires bf16 support, so SM arch must be 80 or
+    # higher.
+    def skipTestForOldSm(self, device: torch.device):
+        if not sm_is_or_higher_than(device, 8, 0):
+            self.skipTest("bf16 requires sm >= 8.0")
```

Comment:
I believe only the tests that have "_inductor" in the name will need to skip bf16, also would be nice if we can use decorator on top of the unit test as well

Reply:
Can you rely on the A10g config to test this TestClass?
I prefer 0-or-1 for the whole TestClass here.